cassdegrees/templates/list.html

```diff
@@ -101,6 +108,15 @@
                 {% endif %}
             </form>
         </div>
+
+        <script>
+            // Creates a list object to be used by list.js for sorting
+            var {{ title }}Options = {
+                valueNames: [{% for k,v in content.0.items %}'{{ title }}{{ k }}',{% endfor %}]
+            };
+
+            var {{ title }}List = new List("{{ title }}Sorter", {{ title }}Options);
```

Comment:
This would break with spaces in titles, no?

Reply:
So `{{ title }}` is always a django relation name. To my knowledge, these can't have spaces, but even if they could there should only be "Program", "Subplan", "Course", so I don't think its such a huge issue. However, I can do it easily by going `{{ title|cut:" " }}`, but IMO this might make it less readable. Thoughts?